tutorials/02-conda_envs.rst

```diff
@@ -0,0 +1,114 @@
+An Astropy User's Guide to Managing Conda Environments and Jupyter Notebook
+===========================================================================
+
+
+    Help! My code library works in Python 2 and I don't have time to
+    make it work in Python 3
+
+Do not fear! This tutorial will teach you how to use conda environments
+to create a *separate* installation of Python 2.7. With conda
+environments, you can switch between Python 2 and 3 without having to
+worry about version conflicts.
+
+Step 1: Set up a Python 2 environment
+-------------------------------------
+
+-  Create a new ``conda`` environment
+
+   ::
+
+       conda create -n python2 python=2.7 anaconda 
+
+**NOTE:** ``anaconda`` at the end is optional...
+
+-  Activate the Python 2 environment and install any additional packages
+   you need to run your favorite code library. Here, we show you how to 
+   install the base version of Astropy. If you want to download
+   Astropy and **all** affiliated packages, you can install ``stsci``
```

Comment:
What all is in `stsci`? 

If we want an astropy "official" metapackage for all of the affiliated packages and their dependencies we can construct that (fairly) painlessly.

Reply:
`stsci` is a metapackage from STScI -- https://github.com/astroconda/astroconda-contrib/blob/master/stsci/meta.yaml

It also installs two other metapackages:
https://github.com/astroconda/astroconda-contrib/blob/master/stsci-hst/meta.yaml
https://github.com/astroconda/astroconda-contrib/blob/master/stsci-data-analysis/meta.yaml

Reply:
I'm not going to include these details in the document.